.github/workflows/cypress.yml

```diff
@@ -0,0 +1,16 @@
+name: cypress-test-run
+on: [push, pull_request]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: npm install
+      - name: Component tests
+        run: npm run test:ct
+      - name: Jest tests
+        run: npm run test
```

Comment:
I would like to have these steps swapped their order. Cypress tests are more heavy to run, so it will provide a bit of optimization: if jest tests fail, then it won't even run cypress ones.